unicorecmsgem/templates/base.pt

```diff
@@ -41,5 +41,9 @@
         <div metal:define-slot="content"></div>
     </div>
     <div id="base"><img alt='' src="/static/img/base.jpg" /></div><!--/BASE-->
+    <div tal:condition="is_sajan_locale()">
+          Translated by: Sajan
+
+    </div>
```

Comment:
If this is always going to be `True` then why are we adding the `tal:condition` instead of just sticking it in the template?